pages/api/payment-received.ts

```diff
@@ -73,6 +120,20 @@ const paymentReceivedHandler: NextApiHandler = async (req, res) => {
     const applicationEmail = application.email;
     const applicationFirstName = application.firstName;
 
+    // Parse billing information
+    const rawBillingInformation = req.body.billing_address;
+    const billingInformation = rawBillingInformation
+      ? {
+          billingFullName: rawBillingInformation.name,
+          billingAddressLine1: rawBillingInformation.address1,
+          billingAddressLine2: rawBillingInformation.address2,
+          billingCity: rawBillingInformation.city,
+          billingProvince: rawBillingInformation.province_code,
+          billingCountry: rawBillingInformation.country,
+          billingPostalCode: stripPostalCode(rawBillingInformation.zip),
+        }
+      : {};
```

Comment:
Do we need to update billing info? From previous discussions i though we wanted source of truth to come from RCD platform. If we do, should we also be updating shipping?

Reply:
Good point. I double-checked the Shopify webhook payload which actually has the billing info, so I think it's reasonable to use this billing address. RCD can actually modify any payment information on any application (they need to manually correct applicant mistakes), so I feel that we can update billing info here.

Regarding shipping info, that is not present in the Shopify form or the payload, so we default to the home address, which is what RCD expects.